### PR TITLE
controls: complete documentaion of [Library],GoToItem

### PR DESCRIPTION
**Pull request by ronso0**
https://deploy-preview-467--mixxx-manual.netlify.app/chapters/appendix/mixxx_controls.html#control-[Library]-GoToItem

### CODE REVIEW

File: source/chapters/appendix/mixxx_controls.rst [hunk 1]
```diff
@@ -3198,7 +3198,7 @@ Note that :mixxx:coref:`[Library],MoveUp` and other Move and Scroll controls emu
    ------------------------  -------------------------------------------------------------------------------------------------------------
    leaf node                 moves focus to tracks table
    ------------------------  -------------------------------------------------------------------------------------------------------------
-   **Tracks table**          loads selected track. See "Playing track protection" in Preferences ‣ Decks
+   **Tracks table**          Performs the "Track Double-Click Action" selected in Preferences ‣ Library. Default: load selected track. Also see "Playing track protection" in Preferences ‣ Decks
```
Comment (Holzhaus) [inline source/chapters/appendix/mixxx_controls.rst:3201]: Please use `guilabel` role instead of inserting triangles manually

Comment (Holzhaus) [reply source/chapters/appendix/mixxx_controls.rst:3201]: Or rather `:menuselection:` as documented here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html

Comment (ronso0) [reply source/chapters/appendix/mixxx_controls.rst]: Yes, I used :menuselection: and also adjusted the existing menu hint.

**Review by Holzhaus — APPROVED** [review]
Thanks!

**Pull request merged**